docs-ui/stories/components/sidebarSection.stories.js

```diff
@@ -0,0 +1,92 @@
+import styled from '@emotion/styled';
+
+import {KeyValueTable, KeyValueTableRow} from 'app/components/keyValueTable';
+import {Body, Main, Side} from 'app/components/layouts/thirds';
+import QuestionTooltip from 'app/components/questionTooltip';
+import SidebarSection from 'app/components/sidebarSection';
+import Tag from 'app/components/tag';
+import TextOverflow from 'app/components/textOverflow';
+import Tooltip from 'app/components/tooltip';
+import space from 'app/styles/space';
+
+export default {
+  title: 'Views/Sidebar Section',
+};
+
+export const Default = () => (
+  <Body>
+    <Main />
+    <Side>
+      <Container>
+        <SidebarSection title="Left Group">{'16 hours'}</SidebarSection>
+        <SidebarSection title="Right Group">{'16 hours'}</SidebarSection>
+      </Container>
```

Comment:
I think this is not really needed because this is really showing off the styling of `<Container>` rather than the functionality of `<SidebarSection>`.